src/core/io/platform/f3xx/ADCf3xx.cpp

```diff
@@ -7,10 +7,10 @@
 #include <HALf3/stm32f3xx_hal_adc.h>
 #include <HALf3/stm32f3xx_hal_adc_ex.h>
 #include <core/platform/f3xx/stm32f3xx.hpp>
+#include <core/utils/log.hpp>
 
 namespace {
-/// This is made as a global variable so that it is accessible in the
-// interrupt.
+/// This is made as a global variable so that it is accessible in the interrupt.
```

Comment:
This anonymous namespace should be in the `core::io` namespace and include the ISR

Reply:
i thought the whole point was that the namespace was outside of the core::io? your saying it should be 
`namespace core::io {`
`namespace {`
`extern "C" void DMA1_Channel1_IRQHandler() {}`
`}`
`... other code here ...`
`}`